src/index.spec.js

```diff
@@ -1,27 +1,222 @@
+import mongodb from 'mongodb';
```

Comment:
I think that using mocks is a better option for unit tests. 
After this is passing just use the same tests for integration tests against a live mongo db.
That way when there is an update on mongodb versions and they break some kind of logic you were counting on, you will know that it is that.